### PR TITLE
Fun with Jest Snapshots 🛑 [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, 

I was playing with snapshots testing, I think that it is very useful in many cases (although I am pretty sure that it is easy to use them in a wrong way).

I am just sharing this PR with you to allow you to know them, I think that we should not merge it (neither the new dependencies) if we do not have real scenarios where apply this tool.

I come to here reading this article: https://kentcdodds.com/blog/effective-snapshot-testing

### CODE REVIEW

**Comment by andresmoschini** [comment]
An interesting article related: https://kentcdodds.com/blog/effective-snapshot-testing

**Comment by andresmoschini** [comment]
I really prefer inline snapshots, what do you think?

File: src/fun-with-snapshots.test.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        Object {
+          "a": 1,
+          "b": 3,
+          "c": 2,
+        }
+      `);
+    });
+
+    it('should include properties with undefined value', async () => {
+      const a = { a: 1, b: 2, c: 3, d: undefined };
```
Comment (mvillaseco) [inline src/fun-with-snapshots.test.js]: WOW,  nice!

**Comment by cbernat** [comment]
> I really prefer inline snapshots, what do you think?

I agree, they are more readable.

**Comment by andresmoschini** [comment]
test this please

**Comment by andresmoschini** [comment]
test this please

**Comment by andresmoschini** [comment]
test this please

**Pull request closed without merging**